Makefile

```
@@ -357,18 +357,25 @@ GATEWAY_API_RELEASE_CHANNEL ?= experimental
 GATEWAY_API_PACKAGE ?= sigs.k8s.io/gateway-api
 GATEWAY_API_CRDS_LOCAL_PATH = $(shell go env GOPATH)/pkg/mod/$(GATEWAY_API_PACKAGE)@$(GATEWAY_API_VERSION)/config/crd
 GATEWAY_API_REPO ?= github.com/kubernetes-sigs/gateway-api
+RAW_GATEWAY_API_REPO ?= https://raw.githubusercontent.com/kubernetes-sigs/gateway-api
```

```suggestion
GATEWAY_RAW_API_REPO ?= https://raw.githubusercontent.com/kubernetes-sigs/gateway-api
```

Maybe this will be consistent with other variable names in this block here?